components/button/button.scss

```diff
@@ -46,6 +46,23 @@
   }
 }
 
+.ui-button_variation_ghost {
```

Comment:
Not according to UI (last row):

![image](https://user-images.githubusercontent.com/595300/27424819-a92095ca-5736-11e7-96c9-b7a77618ea01.png)